python/test/unit/hopper/test_gemm.py

```diff
@@ -115,6 +119,11 @@ def test_gemm_no_scf(M, N, K, NUM_CTAS, NUM_WARPS, TRANS_A, TRANS_B, OUTPUT_TYPE
     else:
         c = torch.empty((M, N), device=a.device, dtype=torch.float32)
 
+    # input "num_ctas" is specific for nvidia backend
+    kwargs = {}
+    if not is_hip():
```

Comment:
Why don't we just pytest.skip tests when num_cta > 1?

Reply:
It seems like we tested more cases by ignoring the `num_ctas` input?

Reply:
I also tried you suggestion, skipping test with num_ctas > 1 also pass.

Reply:
> It seems like we tested more cases by ignoring the `num_ctas` input?

If it's ignored, the same tests are tested for different `num_ctas` values. But they are duplicated.
If skip `num_ctas`>1, only tests with `num_stages == 1` are tests.

Reply:
No, the `num_ctas` is not set in that way that each `num_ctas` value combines with all input shapes.

Reply:
So you want to test all configs here with NUM_CTAS set to 1 on hip. Then it makes sense to ignore instead of skipping.

Reply:
Actually is it useful to test those addtional combinations? Those with non-1 `NUM_CTAS` is meant to be so; if we are swapping to use `NUM_CTAS` to be 1, not sure it's a footgun eventually..

Reply:
OK, I will skip the case with 'NUM_CTAS > 1`. Thanks